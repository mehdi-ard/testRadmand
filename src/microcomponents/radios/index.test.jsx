import React from 'react';
import renderer from 'react-test-renderer';
import Radios from '.';

class RadiosGroup extends React.Component {
  state = {
    data: [
      { title: 'A', id: 1 },
      { title: 'B', id: 2 },
      { title: 'C', id: 3 }
    ]
  };

  render() {
    const { data } = this.state;
    return (
      <>
        <Radios data={data} />
      </>
    );
  }
}

it('renders default', () => {
  expect(renderer.create(<RadiosGroup />).toJSON()).toMatchSnapshot();
});
