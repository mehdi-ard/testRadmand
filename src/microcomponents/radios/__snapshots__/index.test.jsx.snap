// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default 1`] = `
<div
  className="ant-radio-group ant-radio-group-outline"
>
  <label
    className="ant-radio-wrapper"
    style={
      Object {
        "display": "block",
        "lineHeight": "30px",
      }
    }
  >
    <span
      className="ant-radio"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value={1}
      />
      <span
        className="ant-radio-inner"
      />
    </span>
    <span>
      A
    </span>
  </label>
  <label
    className="ant-radio-wrapper"
    style={
      Object {
        "display": "block",
        "lineHeight": "30px",
      }
    }
  >
    <span
      className="ant-radio"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value={2}
      />
      <span
        className="ant-radio-inner"
      />
    </span>
    <span>
      B
    </span>
  </label>
  <label
    className="ant-radio-wrapper"
    style={
      Object {
        "display": "block",
        "lineHeight": "30px",
      }
    }
  >
    <span
      className="ant-radio"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value={3}
      />
      <span
        className="ant-radio-inner"
      />
    </span>
    <span>
      C
    </span>
  </label>
</div>
`;
